app/components/navbar/navbar.component.js

```diff
@@ -4,21 +4,29 @@ import './navbar.scss';
 const navbarComponent = {
   bindings: {},
   template: `
-		<md-toolbar class="navbar">
-			<div class="md-toolbar-tools">
-				<div flex layout="row" layout-align="start center">
-					<img class="ng-hotrod-logo" ng-src="{{ $ctrl.ngHotrodLogo }}" alt="GitHub">
-					<h2 class="active-page-name" md-truncate>{{ $ctrl.NavigationService.currentPageTitle }}</h2>
-				</div>
-				<a 
-					class="github-link" href="https://github.com/xlts-dev/angularjs-material-vite" 
-					layout="row" 
-					layout-align="center center">
-					<img ng-src="{{ $ctrl.githubLogo }}" alt="GitHub">
-					<div>GitHub</div>
-				</a>
-			</div>
-		</md-toolbar>
+	<md-toolbar class="navbar">
+	  <div class="md-toolbar-tools">
+		<div flex layout="row" layout-align="start center">
+	      <img
+            data-testid="ng-hotrod-logo"
+            class="ng-hotrod-logo"
+            ng-src="{{ $ctrl.ngHotrodLogo }}"
+            alt="GitHub"
```

Comment:
Not directly related to this PR (since it was like this before), but this alt text looks wrong 🤔

Reply:
This was not part of my changes 😄 But I think we can leave the alt empty in this case. @gkalpak what do you think?

Reply:
I agree. The image is decorative.

Reply:
Removed